timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -893,6 +893,120 @@ def evaluate(
             logger.info(json.dumps(scores_dict, indent=4))
         return scores_dict
 
+    def feature_importance(
+        self,
+        data: Optional[Union[TimeSeriesDataFrame, pd.DataFrame, Path, str]],
+        model: Optional[str] = None,
+        metric: Optional[Union[str, TimeSeriesScorer]] = None,
+        features: Optional[List[str]] = None,
```

Comment:
Note: Can support feature-group-importance if using Tabular's logic: https://github.com/autogluon/autogluon/blob/master/tabular/src/autogluon/tabular/predictor/predictor.py#L2851-L2857

Reply:
I'll punt if you don't think this is critical.